google-cloud-storage/samples/acceptance/buckets_test.rb

```diff
@@ -349,6 +350,23 @@
     end
   end
 
+  describe "Get Bucket Class And Location data" do
```

Comment:
Make all of them small case. Follow the above pattern, in that way you will not be doing anything different.